app/helpers/blacklight/component_helper_behavior.rb

```diff
@@ -37,6 +37,10 @@ def document_actions(document, options: {})
       filter_partials(blacklight_config.view_config(:show).document_actions, { document: document }.merge(options)).map { |_k, v| v }
     end
 
+    def header_actions(document, options: {})
```

Comment:
Is there a reason we need this helper?  Can we just have it in the component?  We've been trying to move away from helper methods.

Reply:
Done.